api.js

```diff
@@ -3,7 +3,11 @@ const API_KEY = require('./config.js');
 
 let getData = (url, param, callback) => {
   let options = {
-    url: `https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp${url}${param}`,
+    // url: `https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp${url}${param}`,
```

Comment:
**Reason for change:** the get request I need to make includes a path variable, represented in url as '/products/:product_id'. Where '**product_id**' is a **path variable**, not a query parameter. And, my request does not have a query parameter that needs to be passed in. Therefore, I changed the options to work for both scenarios.

I left the original one commented out.